apps/notification-tester/targets/notification-service/Info.plist

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can this be used as it is in every project? Or should I change something?

Reply:
use as-is